src/Forms/HTMLEditor/HTMLEditorSanitiser.php

```diff
@@ -17,8 +18,22 @@
  */
 class HTMLEditorSanitiser
 {
+    use Configurable;
     use Injectable;
 
+    /**
+     * rel attribute to add to link elements which have a target attribute (usually "_blank")
+     * this is to done to prevent reverse tabnabbing - see https://www.owasp.org/index.php/Reverse_Tabnabbing
+     * noopener includes the behaviour we want, though some browsers don't yet support it and rely
+     * upon using noreferrer instead - see https://caniuse.com/rel-noopener for current browser compatibility
+     *
+     * @var string
+     */
+    private static $link_noopener_value = 'noopener, noreferrer';
+
+    /** @var bool - allow users to opt out of this behaviour if required */
+    private static $link_noopener_enabled = true;
```

Comment:
I agree with @NightJar that we don't need this, we can use null in the other config prop to remove it instead - it's a pretty common pattern in SS config to use nullable types and handle them in the implementation

Reply:
have removed and change it so that $link_rel_value being set to null disables behaviour